examples-preview-service/start.sh

```diff
@@ -1,4 +1,9 @@
-preview_path=$(pwd)
+#preview_path=$(pwd)
```

Comment:
```suggestion
preview_path=$(pwd)
```
下面会用到这个变量，注释掉会有问题
![image](https://user-images.githubusercontent.com/26001097/194796269-c31fc362-7f75-48ce-8394-f72199107793.png)